package.json

```diff
@@ -7,7 +7,7 @@
   "scripts": {
     "test": "standard && nyc node test.js",
     "test-browsers-local": "airtap --verbose test-browser.js",
-    "coverage": "nyc report --reporter=text-lcov | coveralls",
```

Comment:
Can you run `npm rm coveralls --save-dev`? LGTM otherwise. Thanks!

Reply:
Removed in 855a208